system_test/arrow_round_trip_fuzz.py

```
@@ -0,0 +1,284 @@
+        if not _round_trip_capable(spec):
+            self.skipTest(f"kind {kind_name!r} not round-trip capable")
+        modes = ["valid", "edge"]
+        if spec.supports_server_null:
+            modes[1:1] = ["partial", "all_null"]
+        for null_mode in modes:
+            with self.subTest(null_mode=null_mode):
+                table = self.fresh_table(f"arrow_rt_{kind_name}_{null_mode}")
+                kinds = [(f"c_{kind_name}", spec)]
+                afc.create_table_from_kinds(self._fixture, table, kinds)
+                ts_base = 1_700_000_000_000_000 + self._master_rng.next_int(1_000_000)
+                rb_in, vpc = _build_batch(
```

Ruff (0.15.14)

[warning] Unpacked variable `vpc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)